packages/components/src/theme/ComposerTheme/Components/Textarea.ts

```diff
@@ -0,0 +1,57 @@
+import { ComponentStyleConfig } from "@chakra-ui/react";
+import { inputStyles } from "../styles";
+
+const baseStyle = {
+  backgroundColor: 'background-action-default',
+  textStyle: "text-body-meta-regular",
+  color: 'text-primary',
+  _placeholder: {
+    color: 'text-secondary',
+  },
+  _hover: {
+    backgroundColor: 'background-action-hover',
+    borderWidth: '0.063rem',
+    borderColor: 'ui-element-outline-active',
+  },
+  _disabled: {
+    backgroundColor: 'background-action-disabled',
+    borderColor: 'ui-element-outline-disabled',
+    _placeholder: {
+      color: 'text-disabled'
+    }
+  },
+  _focus: {
+    backgroundColor: 'background-action-default',
+    borderColor: 'ui-element-outline-default',
+    boxShadow: '0 0 0 0.188rem var(--chakra-colors-primary-focus)',
+  },
+  _invalid: {
+    borderColor: 'alert-error-default'
+  }
+}
+
+const textStyles = {
+  medium: 'text-body-regular',
+  small: 'text-body-meta-regular',
+};
+
+const calculateSize = (size: string) => {
+  return ({ theme }) => ({
+    ...inputStyles[size],
+    ...theme.textStyles[textStyles[size]],
+  })
+}
+
+export const Textarea: ComponentStyleConfig = {
+  baseStyle,
+  variants: {
+    outline: baseStyle
```

Comment:
is this needed?

Reply:
yes, If I remove this property, the styles on focus event don't work:
![image](https://user-images.githubusercontent.com/11969862/151837561-d53e5591-1d97-4864-b512-6746c9f69e88.png)